cmd/internal/codegen/parse/crd.go

```diff
@@ -173,6 +174,24 @@ var primitiveTemplate = template.Must(template.New("map-template").Parse(
     {{ if .Format -}}
     Format: "{{ .Format }}",
     {{ end -}}
+    {{ if .EnumValue -}}
```

Comment:
Add a // TODO on the variable to type check that the enum values match the type of the field - e.g. if the field is numberic the values should parse to be numeric.

Reply:
Good. I have added //TODO comment on `EnumValue` and also implemented a numeric / string type check for parsing enum elements in **getValidation()** func